sync/config/pipelines.yaml

```diff
@@ -26,6 +26,16 @@ archive: https://github.com/tektoncd/pipeline/tags
 #       header: <dict>         # optional, no header added if not set
 #         See https://www.docsy.dev/docs/adding-content/navigation/#section-menu
 tags:
+- name: release-v0.60.x
+  displayName: v0.60.x-LTS
```

Comment:
I don't think v0.60 is an LTS release.
The previous release, v0.59 last month, was an LTS release.

Reply:
We should still add the docs for v0.60, just without the LTS label in the displayName

Reply:
Thanks @AlanGreene  updated.